testsuite/MDAnalysisTests/core/util.py

```diff
@@ -545,3 +546,67 @@ def wrapped_coords(self, compound, center):
         positions = relpos * np.array([a, a, a])
 
         return positions.astype(np.float32)
+
+    def center(self, compound):
+        """Returns centers which correspond to the unwrapped system.
+
+        Parameters
+        ----------
+        compound : {'atoms', 'group', 'segments', 'residues', 'molecules', \
+                    'fragments'}
+            Which type of component is unwrapped. Note that for ``'group'``,
+            the result will only be correct *if the group is the entire system*.
+
+        Note
+        ----
+        This function assumes that all atom masses are equal. Therefore, the
+        returned coordinates for ``center='com'`` and ``center='cog'`` are
+        identical.
+        """
+
+        relpos = self.unwrapped_coords(compound, reference=None)
+
+        comp = compound.lower()
+        if comp not in ['group', 'segments', 'residues', 'molecules',
+                        'fragments']:
+            raise ValueError("Unknown unwrap compound: {}".format(compound))
+
+        pos = 0
+
+        if compound=="residues":
+            center_pos = np.zeros((15, 3), dtype=np.float32)
+        else:
+            center_pos = np.zeros((12, 3), dtype=np.float32)
+
+        for base in range(3):
+            loc_centre = relpos[base, :]
```

Comment:
It's not wrong, but I think we use American English throughout the code base. So use 'center' instead of 'centre'.

Reply:
Yes, sorry...